### ISSUE TITLE
test suite re-runs every migration every time it runs

### ISSUE THREAD

**Issue by quartzjer** [LockerProject/Locker#459]
check the log, it's very hard to debug when something goes wrong and could be causing weird behaviors

**Comment by ctide** [comment]
I wouldn't do anything to cause the test suite to not run migrations, since they could be run by people who have lockers installed.  If we don't think a migration will ever be run again, and we think it has the potential to cause problems, let's remove it.

**Comment by quartzjer** [comment]
Yeah, that's a much better idea, all of them are probably old now.

**Comment by amcjen** [comment]
This should probably be reconsidered as our test suite is being refactored.  (Assigning to Kristjan so it's on his radar!)

**Comment by kristjan** [comment]
I don't know anything relevant yet, but we could compromise by leaving migrations in but having a schema snapshot we kick off tests from. That way we have everything there for those managing a locker but don't incur the init cost during CI.
